### PR TITLE
Add data for :read-only and :read-write selector

### PR DESCRIPTION
**Pull request by ai**
Add by MDN data.

@Fyrd this R is important fo next Autoprefixer release https://github.com/postcss/autoprefixer/issues/543#issuecomment-149341212

### CODE REVIEW

File: features-json/css-read-only-write.json [hunk 1]
```diff
@@ -0,0 +1,250 @@
+{
+  "title":"CSS :read-only and :read-write selectors",
+  "description":":read-only and :read-write pseudo-classes to match elements which are considered user-alterable",
+  "spec":"http://www.w3.org/TR/html5/disabled-elements.html#selector-read-write",
+  "status":"cr",
+  "links":[
+    {
+      "url":"https://css-tricks.com/almanac/selectors/r/read-write-read/",
+      "title":"CSS Trick article"
+    }
```
Comment (cvrebert) [inline features-json/css-read-only-write.json]: Missing a comma here

File: features-json/css-read-only-write.json [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+  "notes":"",
+  "notes_by_num":{
+    "1":"Suppports selector only for input and textarea fields, but not for contenteditable",
+  },
+  "usage_perc_y":60.98,
+  "usage_perc_a":0,
+  "ucprefix":false,
+  "parent":"",
+  "keywords":"css,selector,read-only,read-write",
+  "ie_id":"",
```
Comment (cvrebert) [inline features-json/css-read-only-write.json]: Should be [`cssmutabilitypseudoclasses`](https://dev.modern.ie/platform/status/cssmutabilitypseudoclasses/)

File: features-json/css-read-only-write.json [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    }
+    {
+      "url":"https://developer.mozilla.org/en-US/docs/Web/CSS/%3Aread-only",
+      "title":"MDN :read-only"
+    },
+    {
+      "url":"https://developer.mozilla.org/en-US/docs/Web/CSS/:read-write",
+      "title":"MDN :read-write"
+    },
+    {
```
Comment (cvrebert) [inline features-json/css-read-only-write.json]: No need for this if you set `ie_id`

File: features-json/css-read-only-write.json [hunk 4]
```diff
@@ -0,0 +1,250 @@
+{
+  "title":"CSS :read-only and :read-write selectors",
+  "description":":read-only and :read-write pseudo-classes to match elements which are considered user-alterable",
+  "spec":"http://www.w3.org/TR/html5/disabled-elements.html#selector-read-write",
+  "status":"cr",
+  "links":[
+    {
+      "url":"https://css-tricks.com/almanac/selectors/r/read-write-read/",
+      "title":"CSS Trick article"
```
Comment (cvrebert) [inline features-json/css-read-only-write.json]: Should be "Tricks", not "Trick"

**Comment by ai** [comment]
@cvrebert I fixed all mistakes and make commit again

File: features-json/css-read-only-write.json [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+      "10":"u",
+      "11":"u"
+    },
+    "and_uc":{
+      "9.9":"u"
+    }
+  },
+  "notes":"",
+  "notes_by_num":{
+    "1":"Suppports selector only for input and textarea fields, but not for contenteditable",
```
Comment (Fyrd) [inline features-json/css-read-only-write.json]: Trailing comma here is causing the json to be invalid. (in the future I suggest using http://jsonlint.com to check for json errors)

**Comment by ai** [comment]
@Fyrd fixed. Sorry for quick PR :D. Not I check JSON and it is valid.

**Comment by Fyrd** [comment]
Thanks!

**Comment by ai** [comment]
Awesome! Wait for release.

**Comment by ai** [comment]
@Fyrd what I can do to release it quicker? :)

**Comment by Fyrd** [comment]
@ai I just tested a couple of browsers and your data seems incorrect in various places:
- Firefox should have partial support because it doesn't consider disabled input & textarea fields as readonly
- Opera 12.1 (and probably older ones too) appear to have partial support
- Android 4.4 (and probably older) has partial support

Basically it looks like all the data needs to be reviewed. You cannot just trust MDN data and port it over to caniuse like this. If you want it released, take the time to write tests and do the browser testing yourself. I don't mind doing part of it, but since #1976 has only one vote I don't intend to make it a priority to do a full review over the higher-voted features at http://caniuse.com/issue-list/ .

I understand how just the prefixed data is all you really need for autoprefixer, but rushing partially incorrect data into caniuse is not the way to do it.

**Comment by ai** [comment]
@Fyrd maybe we can hide this data from caniuse website? I think there is a field for it in JSON.

**Comment by Fyrd** [comment]
Yes, all new support data is hidden by default already.

Ah, you're waiting for it to appear in the data.json files, let me take care of that.

**Comment by Fyrd** [comment]
Okay, should be there now. I'd still recommend against using hidden data since, as in this case, it may not all be correct.

**Comment by ai** [comment]
Thanks :)

**Comment by cvrebert** [comment]
See also https://github.com/w3c/csswg-drafts/issues/127

**Pull request merged**